code/modules/mob/living/carbon/brain/life.dm

```diff
@@ -52,104 +52,46 @@
 		adjustFireLoss(5*discomfort)
 
-
-/mob/living/carbon/brain/handle_regular_status_updates()	//TODO: comment out the unused bits >_>
-
-	if(stat == DEAD)
-		eye_blind = max(eye_blind, 1)
-		silent = 0
-	else
-		updatehealth()
-		if( !container && (health < config.health_threshold_dead || ((world.time - timeofhostdeath) > config.revival_brain_life)) )
+/mob/living/carbon/brain/update_stat()
+	if(status_flags & GODMODE)
+		return
+	if(stat != DEAD)
+		if(health <= config.health_threshold_dead || !container)
 			death()
-			eye_blind = max(eye_blind, 1)
-			silent = 0
 			return
+		if(emp_damage>15)
+			if(stat == CONSCIOUS)
+				stat = UNCONSCIOUS
+				set_blindness(1)
 		else
-			stat = CONSCIOUS
-
-			//Handling EMP effect in the Life(), it's made VERY simply, and has some additional effects handled elsewhere
-			if(emp_damage)			//This is pretty much a damage type only used by MMIs, dished out by the emp_act
-				if(!(container && istype(container, /obj/item/device/mmi)))
-					emp_damage = 0
-				else
-					emp_damage = round(emp_damage,1)//Let's have some nice numbers to work with
-				switch(emp_damage)
-					if(31 to INFINITY)
-						emp_damage = 30//Let's not overdo it
-					if(21 to 30)//High level of EMP damage, unable to see, hear, or speak
-						eye_blind = max(eye_blind, 1)
-						setEarDamage(-1,1)
-						silent = 1
-						if(!alert)//Sounds an alarm, but only once per 'level'
-							emote("alarm")
-							src << "<span class='danger'>Major electrical distruption detected: System rebooting.</span>"
-							alert = 1
-						if(prob(75))
-							emp_damage -= 1
-					if(20)
-						alert = 0
-						eye_blind = 0
-						setEarDamage(-1,0)
-						silent = 0
-						emp_damage -= 1
-					if(11 to 19)//Moderate level of EMP damage, resulting in nearsightedness and ear damage
-						eye_blurry = 1
-						setEarDamage(1,-1)
-						if(!alert)
-							emote("alert")
-							src << "<span class='danger'>Primary systems are now online.</span>"
-							alert = 1
-						if(prob(50))
-							emp_damage -= 1
-					if(10)
-						alert = 0
-						eye_blurry = 0
-						setEarDamage(0,-1)
-						emp_damage -= 1
-					if(2 to 9)//Low level of EMP damage, has few effects(handled elsewhere)
-						if(!alert)
-							emote("notice")
-							src << "<span class='danger'>System reboot nearly complete.</span>"
-							alert = 1
-						if(prob(25))
-							emp_damage -= 1
-					if(1)
-						alert = 0
-						src << "<span class='danger'>All systems restored.</span>"
-						emp_damage -= 1
-			else
-				eye_blind = 0
-
-		return 1
+			if(stat == UNCONSCIOUS)
+				stat = CONSCIOUS
+				set_blindness(0)
 
-/mob/living/carbon/brain/handle_disabilities()
-	//Eyes
-	if(disabilities & BLIND || stat)
-		eye_blind = max(eye_blind, 1)
-	else
-		if(eye_blind)
-			eye_blind = 0
-		if(eye_blurry)
-			eye_blurry = 0
-		if(eye_stat)
-			eye_stat = 0
-
-	//Ears
-	if(disabilities & DEAF)
-		setEarDamage(-1, max(ear_deaf, 1))
-	else if(ear_damage < 100)
-		setEarDamage(0, 0)

```

Comment:
unneeded brain can be blinded or its eyes and ear damaged. The only possible blindness is through emp damage and its handled in brain/update_stat()